BlueDBInMemory/src/main/java/io/bluedb/memory/BlueCollectionImpl.java

```diff
@@ -107,6 +109,7 @@ public void updateAll(long minTime, long maxTime, List<Condition<T>> objectCondi
 				results.add(key);
 			}
 		}
+		sort(results);

```

Comment:
Is it important to delete and update in order? Doesn't really matter for the in memory impl, but might on disk.